install/sessions.csv

```diff
@@ -231,7 +233,8 @@ LEVN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2015-09-24T0
 LEVN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2016-12-19T20:44:01Z,2022-01-27T09:34:29Z
 LEVN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2022-01-27T09:34:30Z,2022-03-16T23:59:59Z
 LEVN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2022-03-17T00:00:00Z,2024-07-25T02:08:37Z
-LEVN,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2024-07-25T02:08:38Z,9999-01-01T00:00:00Z
+LEVN,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2024-07-25T02:08:38Z,2025-02-19T22:32:12Z
+LEVN,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2025-02-19T22:32:13Z,9999-01-01T00:00:00Z
```

Comment:
I think we don't need to update the timestamp as the site has been configured to track mulit-GNSS already.